README.md

```diff
@@ -125,6 +102,70 @@ With respect to the `tags` configuration option, the tags that will be added are
 
 If you want a metric to be strictly host-local, you can tell Veneur not to forward it by including a `veneurlocalonly` tag in the metric packet, eg `foo:1|h|#veneurlocalonly`. This tag will not actually appear in DataDog; Veneur removes it.
 
+# Configuration
+
+Veneur expects to have a config file supplied via `-f PATH`. The include `example.yaml` outlines the options:
+
+* `api_hostname` - The Datadog API URL to post to. Probably `https://app.datadoghq.com`.
+* `metric_max_length` - How big a buffer to allocate for incoming metric lengths. Metrics longer than this will get truncated!
+* `flush_max_per_body` - how many metrics to include in each JSON body POSTed to Datadog. Veneur will POST multiple bodies in parallel if it goes over this limit. A value around 5k-10k is recommended; in practice we've seen Datadog reject bodies over about 195k.
+* `debug` - Should we output lots of debug info? :)
+* `hostname` - The hostname to be used with each metric sent. Defaults to `os.Hostname()`
+* `interval` - How often to flush. Something like 10s seems good.
+* `key` - Your Datadog API key
+* `percentiles` - The percentiles to generate from our timers and histograms. Specified as array of float64s
+* `udp_address` - The address on which to listen for metrics. Probably `:8126` so as not to interfere with normal DogStatsD.
+* `http_address` - The address to serve HTTP healthchecks and other endpoints. This can be a simple ip:port combination like `127.0.0.1:8127`. If you're under einhorn, you probably want `einhorn@0`.
+* `forward_address` - The address of an upstream Veneur to forward metrics to. See below.
+* `num_workers` - The number of worker goroutines to start.
+* `num_readers` - The number of reader goroutines to start. Veneur supports SO_REUSEPORT on Linux to scale to multiple readers. On other platforms, this should always be 1; other values will probably cause errors at startup. See below.
+* `read_buffer_size_bytes` - The size of the receive buffer for the UDP socket. Defaults to 2MB, as having a lot of buffer prevents packet drops during flush!
+* `stats_address` - The address to send internally generated metrics. Probably `127.0.0.1:8125` to send to a local DogStatsD
```

Comment:
ah, I forgot to update this line after implementing it, but we should probably recommend that veneur use its own udp address here